tools/packaging/debian/copyright

```diff
@@ -0,0 +1,9 @@
+Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: preCICE
+Upstream-Contact: preCICE mailing-list <precice@mailman.informatik.uni-stuttgart.de>
+Copyright: (c) 2019 the preCICE developers <precice@mailman.informatik.uni-stuttgart.de>
```

Comment:
My previous comment was shown as outdated, which it definitively not is. I think the mail address should be `info@precice.org`